ResearchKit/ActiveTasks/ORKTowerOfHanoiStepViewController.m

```diff
@@ -143,6 +143,10 @@ - (void)towerOfHanoiTowerViewWasSelected:(ORKTowerOfHanoiTowerView *)towerView {
     [self evaluatePuzzle];
 }
 
+- (ORKTowerOfHanoiTowerView *)towerOfHanoiHighlightedTowerView {
+    return (_selectedIndex != nil ? _towerViews[_selectedIndex.integerValue] : nil);
+}
+
 #pragma Mark -- ORKTowerOfHanoiViewController
```

Comment:
I have not reviewed the accessibility changes per-se (busy with the graphs) but I spotted there are several incorrect `#pragma Mark -- ...` lines on the tower of hanoi files. This PR might be a good change to fix those. They should be ` #pragma mark - ...`.